src/configParser.ts

```diff
@@ -136,7 +136,7 @@ export class ConfigParser {
         inputFilePath: this._getCircuitInputPath(inputDir, c),
         outputDir: cktOutputDir,
         powerOfTauFp: path.resolve(
-          c.powerOfTauFp ? c.powerOfTauFp : path.join(inputDir, `power_of_tau.ptau`) // Calculate total constraints from r1cs file, and download it
+          path.join(cktOutputDir, `power_of_tau.ptau`) // Calculate total constraints from r1cs file, and download it
```

Comment:
the path should only be calcuated when the filepath is not supplied. 
In case the user supplies the filepath, no tau file should be downloaded else it should be downloaded inside outputdir/tau/ directory